test/cpp/interop/xds_interop_client.cc

```diff
@@ -438,12 +438,11 @@ void EnableCsmObservability() {
   auto meter_provider =
       std::make_shared<opentelemetry::sdk::metrics::MeterProvider>();
   meter_provider->AddMetricReader(std::move(prometheus_exporter));
-  assert(grpc::OpenTelemetryPluginBuilder()
-             .AddPluginOption(
-                 grpc::experimental::MakeCsmOpenTelemetryPluginOption())
-             .SetMeterProvider(std::move(meter_provider))
-             .BuildAndRegisterGlobal()
-             .ok());
+  auto observability = grpc::experimental::CsmObservabilityBuilder()
+                           .SetMeterProvider(std::move(meter_provider))
+                           .BuildAndRegister();
+  assert(observability.ok());
+  return *observability;
```

Comment:
This makes me realize that we should change `CsmObservability` to be a move-only type.  If we allow it to be copyable, then if we want to add something like a `unique_ptr<>` to it later, we will break existing code.

Reply:
oh wow, that was a really good catch. Thanks Mark!